src/conversion/area/acres.js

```diff
@@ -0,0 +1,11 @@
+const shared = require('../shared');
+const area = require('./_area');
+
+module.exports = {
+    "imperialUnits" : [/acres?/],
+    "standardInputUnit" : " acres",
+    "isInvalidInput" : shared.isZeroOrNegative,
+    "isWeaklyInvalidInput" : shared.isHyperbole,
+    "conversionFunction" : (i) => area.toMap(i, (j) => j * 4046.8564),
```

Comment:
I'm not really sure what value passing a seperate function, and then instantly resolving the value(inside `areaMap`) provides, why not just `i.map((j) => j * 4046.8564)` here itself?

Reply:
I thought it'd be easier if we didn't need to do `.map` at the input level, but instead pushed it down lower. For "area" it's the same amount of work since we are only converting one unit (acre) but for weight, it saves us from typing `.map` 2 times (since there are 3 weight units, oz, troy oz, and lb).

Reply:
Fair enough I suppose, just makes the `areaMap` a bit confusing IMO, but I guess its not that bad.

Reply:
It also allows us to use the imperial inputs as part of our calculations (See _temperature.js - It's easier to calculate "change in degree C" directly from degree F ... also I just noticed a bug in it lacking a "unitDecider", I'll fix that now haha)

Reply:
Fair point, it does make the interface less clear. Hmmm... let's keep it for now, and we can always refactor again if it becomes a problem